build.gradle

```diff
@@ -66,20 +64,9 @@ buildscript {
             "junitTest"                  : "androidx.test.ext:junit:1.1.3",
             "robolectric"                : "org.robolectric:robolectric:4.7.3",
             "dexmakerMockito"            : "com.google.dexmaker:dexmaker-mockito:1.2",
-            "mockitoCore"                : "org.mockito:mockito-core:3.6.0",
+            "mockitoCore"                : "org.mockito:mockito-core:5.3.1",
```

Comment:
Do we need to call this version bump out in our changelog or does this only matter for our internal tests?

Reply:
For mockito it'll be fine. Since we mark it as a test dependency it won't ship when we release.